charts/consul/templates/gateway-resources-job.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
   annotations:
     "helm.sh/hook": post-install,post-upgrade
     "helm.sh/hook-weight": "0"
-    "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
why did we need to update the gateway resources job here?

Reply:
It was not deleted after failed during my test, but now, it should be gone